community/codelabs/docai-ocr/batch_processing.py

```diff
@@ -1,69 +1,32 @@
 # type: ignore[1]
+# pylint: skip-file
```

Comment:
does this cause pylint to skip this whole file ?

Reply:
Yes, Changing to just disable pylint on the line causing issues